flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/operations/SqlToOperationConverterTest.java

```diff
@@ -271,23 +275,78 @@ public void testCreateTable() {
 				DataTypes.VARCHAR(Integer.MAX_VALUE)});
 	}
 
-	@Test(expected = SqlConversionException.class)
-	public void testCreateTableWithPkUniqueKeys() {
-		FlinkPlannerImpl planner = getPlannerBySqlDialect(SqlDialect.DEFAULT);
-		final CalciteParser parser = getParserBySqlDialect(SqlDialect.DEFAULT);
+	@Test
+	public void testCreateTableWithPrimaryKey() {
 		final String sql = "CREATE TABLE tbl1 (\n" +
 			"  a bigint,\n" +
 			"  b varchar, \n" +
 			"  c int, \n" +
 			"  d varchar, \n" +
-			"  primary key(a), \n" +
-			"  unique(a, b) \n" +
-			")\n" +
-			"  PARTITIONED BY (a, d)\n" +
-			"  with (\n" +
-			"    'connector' = 'kafka', \n" +
-			"    'kafka.topic' = 'log.test'\n" +
+			"  constraint ct1 primary key(a, b) not enforced\n" +
+			") with (\n" +
+			"  'connector' = 'kafka', \n" +
+			"  'kafka.topic' = 'log.test'\n" +
 			")\n";
+		FlinkPlannerImpl planner = getPlannerBySqlDialect(SqlDialect.DEFAULT);
+		final CalciteParser parser = getParserBySqlDialect(SqlDialect.DEFAULT);
+		Operation operation = parse(sql, planner, parser);
+		assert operation instanceof CreateTableOperation;
+		CreateTableOperation op = (CreateTableOperation) operation;
+		CatalogTable catalogTable = op.getCatalogTable();
+		TableSchema tableSchema = catalogTable.getSchema();
+		assertThat(
+				tableSchema
+						.getPrimaryKey()
+						.map(UniqueConstraint::asSummaryString)
+						.orElse("fakeVal"),
+				is("CONSTRAINT ct1 PRIMARY KEY (a, b)"));
+		assertArrayEquals(tableSchema.getFieldNames(),
+				new String[] {"a", "b", "c", "d"});
```

Comment:
Swap these 2 parameters, the first parameter should be `expected` , the second should be `actual`.

```
assertArrayEquals(new String[] {"a", "b", "c", "d"}, tableSchema.getFieldNames());
```